src/saving/FileHelpers.cs

```diff
@@ -43,4 +44,22 @@ public static bool Exists(string path)
         using var directory = new Directory();
         return directory.FileExists(path);
     }
+
+    /// <summary>
+    ///   Tests if it is possible to create/write a file on the given path
+    /// </summary>
+    /// <param name="path">Path to file</param>
+    /// <returns>File write access status, <see cref="Error.Ok"/> if successful</returns>
+    public static Error TryCreateWrite(string path)
+    {
+        using var file = new File();
+        var e = file.Open(path, File.ModeFlags.Write);
```

Comment:
e should be named something else. This is not one of the contexts where single letter variables are allowed.